composer.json

```diff
@@ -36,7 +36,7 @@
         "league/flysystem-aws-s3-v3": "AwsS3 adapter support.",
         "league/flysystem-azure": "Azure adapter support.",
         "league/flysystem-cached-adapter": "Adapter caching support.",
-        "league/flysystem-dropbox": "Dropbox adapter support.",
+        "srmklive/flysystem-dropbox-v2": "Dropbox adapter support.",
```

Comment:
Hi Graham,

Yes, league/flysystem-dropbox refers to spatie/flysystem-dropbox, which then refers to srmklive/flysystem-dropbox-v2 for PHP5 support. See: 
https://github.com/spatie/flysystem-dropbox/blob/master/README.md